packages/nss.rb

```diff
@@ -3,70 +3,64 @@
   homepage 'https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS'
-  @_ver = '3.61'
-  version @_ver
+  @nss_ver = '3.69.1'
+  @nspr_ver = '4.32'
+  version "nss.#{@nss_ver}.nspr.#{@nss_ver}"
   license 'MPL-2.0, GPL-2 or LGPL-2.1'
   compatibility 'all'
-  source_url 'https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_61_RTM/src/nss-3.61-with-nspr-4.29.tar.gz'
-  source_sha256 '812468f3cf22917f9647fec7997f4ab27ae4167811d9cbdc831f41f5ed281c5d'
+  source_url 'https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_69_1_RTM/src/nss-3.69.1-with-nspr-4.32.tar.gz'
+  source_sha256 '976dd57391aa269aa2961ac7ae002dc49935f855263df58b59b1dd4836d47cb7'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/3.61_armv7l/nss-3.61-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/3.61_armv7l/nss-3.61-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/3.61_i686/nss-3.61-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/3.61_x86_64/nss-3.61-chromeos-x86_64.tar.xz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/nss.3.69.1.nspr.4.32_armv7l/nss-nss.3.69.1.nspr.4.32-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/nss.3.69.1.nspr.4.32_armv7l/nss-nss.3.69.1.nspr.4.32-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/nss.3.69.1.nspr.3.69.1_i686/nss-nss.3.69.1.nspr.3.69.1-chromeos-i686.tar.xz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/nss/nss.3.69.1.nspr.4.32_x86_64/nss-nss.3.69.1.nspr.4.32-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: '812f36d5c1875cf8803097265483b8a558194c6f0778977d55637407682ba51c',
-     armv7l: '812f36d5c1875cf8803097265483b8a558194c6f0778977d55637407682ba51c',
-       i686: 'e724b986a9a2edce790a8bef0b89d096ff46cb93815fc2d7bf8beb950af851f5',
-     x86_64: 'ba50721b2e968c49fbd5d4a90112ff27374903d33dbcfc3f1877b2c7074adbcd'
+    aarch64: 'e68a620a7b7f35289f401561e3c86ff1fceb38d0b0652da7e6c531addc020a14',
+     armv7l: 'e68a620a7b7f35289f401561e3c86ff1fceb38d0b0652da7e6c531addc020a14',
+       i686: '88d3a49f76d9fe8722aab3f9553a576609741004c2acdd2773bef28b2cc9e12a',
+     x86_64: '89f80e57d77168fe58e28f60e245a18256dd3b1c68640a1430fc512f93e01b01'
   })
 
   depends_on 'gyp_next' => :build
-  depends_on 'nspr'
 
   def self.build
-    ENV['opt_build'] = '1'
-    ENV['build_64'] = if ARCH == 'x86_64'
-                        '1'
-                      else
-                        '0'
-                      end
-    @arch_cflags = if ARCH == 'armv7l'
-                     ''
-                   else
-                     '-flto=auto'
-                   end
+    @build_64 = ARCH == 'x86_64' ? '1' : '0'
+    @arch_cflags = ''
```

Comment:
`@arch_ldflags = @arch_cflags = ''`?

Reply:
Sure? That's debug code at this point, since LTO was previously being used for non-armv7l, but it just causes issues for everything. Maybe it can be revisited for a later revision.

Reply:
(At which point it would likely have to be broken out again if there is an arch where it does work right.)

Reply:
Got it, never mind :)